examples/sentiment/train_recursive_minibatch.py

```diff
@@ -215,18 +211,17 @@ def main():
         test_data, args.batchsize, repeat=False, shuffle=False)
 
     model = ThinStackRecursiveNet(len(vocab), args.unit, args.label)
-
-    if args.gpu >= 0:
-        model.to_gpu()
+    model.to_device(device)
 
     optimizer = chainer.optimizers.AdaGrad(0.1)
     optimizer.setup(model)
 
     updater = training.StandardUpdater(
-        train_iter, optimizer, device=None, converter=convert)
+        train_iter, optimizer, converter=convert, device=device)
```

Comment:
This order of arguments is inconsistent with `train_sentiment.py`.
I have no preference but inconsistency is not good.

Reply:
I changed the order since it was already "inconsistent" with the `Evaluator` constructor below but then let me align all occurrences to this one instead because that's the order of the arguments defined in the constructors of the `StandardUpdater` and `Evaluator`.

Reply:
Fixed.